### PR TITLE
expression: Add format_bytes and format_nano_time performance schema functions

### LINKED ISSUE #14886: Support performance schema functions
## Feature Request

**Is your feature request related to a problem? Please describe:**

Once we have created a view on system tables in performance_schema, the row data are always hard to understand.

For example, 
```
mysql> select AVG_MEM, AVG_LATENCY from events_statements_summary_by_digest limit 1;
+----------+-------------+
| AVG_MEM  | AVG_LATENCY |
+----------+-------------+
| 19074000 |    23338886 |
+----------+-------------+
1 row in set (0.00 sec)
```
One can hardly understand how much memory or time does it really cost.

**Describe the feature you'd like:**

In MySQL, [performance schema functions](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-functions.html) are used to convert these integers into strings that are easy to understand.

The expected result is
```
select format_bytes(AVG_MEM) as AVG_MEM, format_nano_time(AVG_LATENCY) as AVG_LATENCY from events_statements_summary_by_digest limit 1;
+-----------+-------------+
|  AVG_MEM  | AVG_LATENCY |
+-----------+-------------+
| 19.07 MiB |    23.33 ms |
+-----------+-------------+
```

For now, we only need `FORMAT_BYTES` and `FORMAT_NANO_TIME`, as time unit in TiDB is always nanosecond, not picosecond.

**Teachability, Documentation, Adoption, Migration Strategy:**

These functions are commonly used in system views in sys schema.

### PR DESCRIPTION
**Pull request by gauss1314**
### What problem does this PR solve? 

This fix #14886 .
Add performance schema functions: format_bytes and format_nano_time.

### Check List

Tests

 - Unit test

### CODE REVIEW

**Comment by djshow832** [comment]
Since the parser PR has been merged, you can update go.sum and continue now. @gauss1314

**Comment by djshow832** [comment]
UnitTest failed because another PR is not merged yet. I'll push them to merge and let you know when it's done.

File: expression/builtin_info_test.go [hunk 1]
```diff
@@ -276,3 +277,49 @@ func (s *testEvaluatorSuite) TestLastInsertID(c *C) {
...
+func (s *testEvaluatorSuite) TestFormatNanoTime(c *C) {
+	tbl := []struct {
+		Arg interface{}
+		Ret interface{}
+	}{
+		{nil, nil},
+		{float64(0), "0 ns"},
+		{float64(2000), "2.00 us"},
+		{float64(9999999991), "10.00 s"},
+		{float64(-9999999991), "-10.00 s"},
```
Comment (djshow832) [inline expression/builtin_info_test.go]: Please add more test cases to test minutes, hours and days.

**Comment by djshow832** [comment]
The CI problem has been fixed, you can resolve conflicts and run CI again.

**Comment by djshow832** [comment]
Unit test `TestShowBuiltin` failed, please modify it and run `make test` locally to verify that all cases are ok.

File: executor/show_test.go [hunk 2]
```diff
@@ -698,7 +698,7 @@ func (s *testSuite5) TestShowBuiltin(c *C) {
 	res := tk.MustQuery("show builtins;")
 	c.Assert(res, NotNil)
 	rows := res.Rows()
-	c.Assert(266, Equals, len(rows))
+	c.Assert(268, Equals, len(rows))
 	c.Assert("abs", Equals, rows[0][0].(string))
-	c.Assert("yearweek", Equals, rows[265][0].(string))
+	c.Assert("xor", Equals, rows[265][0].(string))
```
Comment (djshow832) [inline executor/show_test.go]:
```suggestion
	c.Assert("yearweek", Equals, rows[267][0].(string))
```

**Comment by gauss1314** [comment]
Hi @djshow832  , maybe I should add some test case in `executor/show_test.go`?

**Comment by djshow832** [comment]
> Hi @djshow832 , maybe I should add some test case in `executor/show_test.go`?

Sure, if you think it's needed.

**Pull request merged**